evansmusomi/burger-builder
# Creates Order Input Fields Dynamically and Handles Submission

*What this does*
- Extends Input Component
- Creates Input Fields Dynamically based on JS Config
- Handles User Input (2-way binding)
- Handles Form Submission

*How to verify*
- User can submit order data successfully after continuing to check out
- ContactData component contains JS config used to create Input Fields
